src/AtomShell/window.jl

```diff
@@ -154,71 +163,116 @@ flashframe(win::Window, on = true) =
   @dot_ win flashFrame($on)
 
 """
-    progress(win::Window, p=-1)
+    progress!(win::Window, p=-1)
 
 Sets progress value in progress bar. Valid range is [0, 1.0]. Remove progress
 bar when progress < 0; Change to indeterminate mode when progress > 1.
 
 https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winsetprogressbarprogress-options
 """
-progress(win::Window, p = -1) =
-  @dot_ win setProgressBar($p)
+progress!(win::Window, p) = @dot_ win setProgressBar($p)
+
+# Deprecated
+progress(win::Window, progress=-1) = progress!(win, progress)
 
 """
-    title(win::Window, title)
+    title!(window, title)
 
-Set `win`'s title to `title.`
+Set the window's title.
 """
-title(win::Window, title) =
-  @dot_ win setTitle($title)
+title!(win::Window, title) = @dot_ win setTitle($title)
+
+# Deprecated
+title(win::Window, title) = title!(win, title)
 
 """
-    title(win::Window)
+    title(window)
 
 Get the window's title.
 """
-title(win::Window) =
-  @dot win getTitle()
+title(win::Window) = @dot win getTitle()
+
+"""
+    center!(win::Window)
+
+Center a window.
+"""
+center!(win::Window) = @dot_ win center()
 
+# Deprecated (and misspelled?)
```

Comment:
haha 😛 Maybe this was written by a british english speaker? Perhaps Shashi? I know he wrote a lot of this originally, right?